core/objlist.hpp

```diff
@@ -281,6 +281,30 @@ class ObjList : public ObjListBase {
         del_bitmap_.swap(tmp_bool);
     }
 
+    size_t estimated_storage_size() {
+        const size_t sample_num = this->get_size()*0.005;
+        BinStream bs;
+
+        // sample
+        std::unordered_set<size_t> sample_container;
```

Comment:
Why we need an unordered_set to store the index? Can't we just put into the BinStream?